builder/builder.go

```diff
@@ -88,11 +89,26 @@ func (b *Builder) ScheduleNewBuildWithContainerFile(workflowName string, imageTa
 }
 
 func (b *Builder) ScheduleNewKanikoBuildWithContainerFile(workflowName string, imageTag string, workflowDefinition []byte, build api.BuildSpec) (*api.Build, error) {
-	ib := b.getImageBuilderForKaniko(workflowName, imageTag, workflowDefinition, build.Tasks[0].Kaniko)
+	task, found := findKanikoTask(build.Tasks)
```

Comment:
this:
```
task, found := findKanikoTask(build.Tasks)
if !found {
	return nil, fmt.Errorf("Unable to find KanikoTask in Build CR for workflow %s", workflowName)
}
```

can be turned in something like this: 
```
if task, found := findKanikoTask(build.Tasks); !found {
	return nil, fmt.Errorf("Unable to find KanikoTask in Build CR for workflow %s", workflowName)
} else {
	ib := b.getImageBuilderForKaniko(workflowName, imageTag, workflowDefinition, task)
	ib.WithTimeout(5 * time.Minute)
	return b.BuildImage(ib.Build())
}
```

Reply:
it does not change nothing, just sabe a few lines :)

Reply:
Saved one line of code ;-)